guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -27,11 +27,15 @@ endif::[]
 
 The following tables indicate the destination port and the direction of network traffic:
 
-.Ports for {SmartProxy} to {Project} Communication
+Ports for {SmartProxy} to {Project} Communication
```

Comment:
just FYI: the dot without space makes this line the title of the table.

Reply:
I wonder if it was meant as a heading? Then perhaps adding a newline after the heading instead of removing the dot?

Reply:
It's the table title.